controls/V-72315.rb

```diff
@@ -20,6 +20,54 @@
 -----------------
 =end
 
+# These attributes must be filled in to reflect expectations of particular system
+FIREWALLD_SERVICES = attribute(
+  'firewalld_services',
+  default: [
+    # Examples
+    # 'dhcpv6-client',
+    # 'ssh'
+  ],
+  description: "Services that firewalld should be configured to allow."
+)
+
+FIREWALLD_HOSTS_ALLOW = attribute(
+  'firewalld_hosts_allow',
+  default: [
+    # Example
+    # 'rule family="ipv4" source address="92.188.21.1/24" accept]'
+  ],
+  description: "Hosts that firewalld should be configured to allow."
+)
+
+FIREWALLD_PORTS_ALLOW = attribute(
+  'firewalld_ports_allow',
+  default: [
+    # Examples
+    # '22/tcp',
+    # '4722/tcp'
+  ],
+  description: "Ports that firewalld should be configured to allow."
+)
+
+TCPWRAPPERS_ALLOW = attribute(
+  'tcpwrappers_allow',
+  default: [
+    # Example
+    # { 'daemon' => 'ALL', 'client_list' => ['ALL'], 'options' => ['allow'] }
+  ],
+  description: "Allow rules from etc/hosts.allow."
+)
+
+TCPWRAPPERS_DENY = attribute(
```

Comment:
These are probably going to fail with 'global already defined' warnings when using inherited profiles. They should not be constants and need to be contained within the `control`

Reply:
I'll change this. There are a few of these hanging out throughout the profile.

Reply:
@trevor-vaughan this is a known issue that I really need to get inspec to fix in terms of how they load and manage attributes in profiles. Perhaps you could reference the need in an issue with this example it would help to have an 'outside source'.